helm-chart/templates/deployment.yaml

```diff
@@ -32,11 +32,29 @@ spec:
                   name: {{ template "emx2.fullname" $ }}
                   key: adminpassword
             - name: MOLGENIS_POSTGRES_USER
-              value: molgenis_admin
```

Comment:
I see you make this more flexible. While it seems nice to be flexible, in my mind we now have dramatically increased the operational cost of running this because now a lot of addtional paramaters must be filled (while also adding the cost maintaining this flexibility). I don't believe we should do this: can we please keep it simple?